style.css

```diff
@@ -1,3 +1,3 @@
 h1 {
-    color: cyan;
```

Comment:
### Optional
- [ ] It would be better if our PR actually describes what we did in this project, for example;
```
- Setup and check linters
- Maintain a professional repository
- GitHub flow correctly used
- Necessary files created
```